scripts/gulp/live-reload-server.js

```diff
@@ -15,8 +15,8 @@ function licenseText() {
   return fs.readFileSync('./LICENSE', 'utf-8');
 }
 
-function changelogText() {
-  return fs.readFileSync('./CHANGELOG.md', 'utf-8');
+function codeOfConductText() {
```

Comment:
Can you help me understand how these changes fit in with this set of changes?

Reply:
The client's dev server used CHANGELOG.md as a dummy document. That's being removed so it needs to render something else instead.